### PR TITLE
Makes practice lasers scarier

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
They look and sound like real lasers

### CODE REVIEW

**Comment by optimumtact** [comment]
Please no :-1: I have trouble distinguishing them already

**Comment by neersighted** [comment]
:+1: smart change

**Comment by Bawhoppen** [comment]
:+1:x:100:

File: code/modules/projectiles/projectile/beams.dm [hunk 1]
```diff
@@ -9,9 +9,8 @@
 	eyeblur = 2
 
 /obj/item/projectile/beam/practice
-	name = "practice laser"
+	name = "laser"
```
Comment (ChangelingRain) [inline code/modules/projectiles/projectile/beams.dm]: beams are already NAMED lasers

**Comment by LanCartwright** [comment]
holy kek i've flat out emptied shotguns into people for practice lasering me
but this

**Comment by Buggy123** [comment]
:+1:

**Comment by Thunder12345** [comment]
Yeah if you practice laser someone with lethals don't be surprised if they return fire

**Comment by TheNightingale** [comment]
The biggest giveaway is firing a "practice laser gun". Maybe have it only show it's fake when examined?

**Comment by Buggy123** [comment]
I agree, rename it to a "laser gun" that just looks different on examine.

**Comment by Bawhoppen** [comment]
now make red laser tag beams just like normal lasers and blue laser tag beams like disablers

**Comment by tkdrg** [comment]
#6737

**Comment by KorPhaeron** [comment]
@tkdrg 

Attack logs now display health changes as well, so it should be identifiable as a fake laser to admins

**Pull request merged**